SCCSFirstLaba/SCCSFirstLaba/Services/ExcelService.cs

```diff
@@ -0,0 +1,39 @@
+﻿using SCCSFirstLaba.Core;
+using SCCSFirstLaba.FileHelpers;
+using SCCSFirstLaba.Interfaces;
+using SCCSFirstLaba.Models;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace SCCSFirstLaba.Services
+{
+    public class ExcelService : IService
+    {
+        private readonly IFileHelper _fileHelper;
+
+        public ExcelService()
+        {
+            _fileHelper = new ExcelHelper();
+        }
+   
+        public void Create(IEnumerable<Student> item, string path)
+        {
+            var dataToWrite = new List<DataToWrite>();
+
+            foreach (var student in item)
```

Comment:
I think Services level is useless here.
You can use AutoMapper or create a additional singlestatic class with mappings that you described here.
Remove all services and create single class with mappings that will be used in your app.